### PR TITLE
Add link to heading for easy access

### LINKED ISSUE #119: Add reference URL to the company heading for quick access
The same reference URL can be added to the heading for easy access, ie, click on the company name would redirect to the same reference link as specified in "Check this out".

### PR DESCRIPTION
**Pull request by RajatArora08**
- [X] I've checked that this isn't a duplicate pull request.

Adds link to heading for easy access

Fixes #119

### CODE REVIEW

**Comment by plibither8** [comment]
I don't think we should do that. It's unintuitive as there won't be anything indicating the presence of a link until the mouse hovers over the card. `a` tags are by default inline elements, and it always seems hack-y to use them as block-level elements. Moreover, users won't be able to select/highlight text in the item description and the links present in the item description.

File: site/src/pug/includes/body.pug [hunk 1]
```diff
@@ -40,7 +40,8 @@ include ./fork.pug
                 - difficultyTitle = 'Difficulty: ' + swag.difficulty;
                 div.item
                     div.title.flex
-                        h1=swag.name
+                        h1
+                          a.no-underline(href=swag.reference) #{swag.name}
```
Comment (plibither8) [inline site/src/pug/includes/body.pug]: Currently, the `h1` color is taking the color of `a` tags. I suggest removing `.no-underline` class
```suggestion
                          a(href=swag.reference) #{swag.name}
```

File: site/src/styl/imports/card.styl [hunk 2]
```diff
@@ -52,6 +52,9 @@
         text-decoration none
         border-bottom 2px solid #444
 
+    .no-underline
```
Comment (plibither8) [inline site/src/styl/imports/card.styl]: `a` tag present in `h1` should be declared under the `h1` styles like so:

```styl
.title
        justify-content space-between

        h1
            font-size 2rem
            
                a
                    border-bottom none
```

**Comment by swapagarwal** [comment]
@RajatArora08 Let's rebase and push this out?

**Comment by aslafy-z** [comment]
Ping @RajatArora08

**Comment by aslafy-z** [comment]
Hello @RajatArora08! Could you please rebase your PR?

File: site/src/pug/includes/body.pug [hunk 3]
```diff
@@ -40,7 +40,8 @@ include ./fork.pug
                 - difficultyTitle = 'Difficulty: ' + swag.difficulty;
                 div.item
                     div.title.flex
-                        h1=swag.name
+                        h1
+                          a(href=swag.reference) #{swag.name}
```
Comment (aslafy-z) [inline site/src/pug/includes/body.pug]:
```suggestion
                          a(href=swag.reference target="_blank") #{swag.name}
```

Comment (aslafy-z) [reply site/src/pug/includes/body.pug]: ping @RajatArora08

File: site/src/styl/imports/card.styl [hunk 4]
```diff
@@ -13,6 +13,9 @@
         h1
             font-size 2rem
 
+            a
+                border-bottom none
```
Comment (aslafy-z) [inline site/src/styl/imports/card.styl:17]:
```suggestion
                text-decoration none
```

**Comment by aslafy-z** [comment]
Hello @RajatArora08. Any chance you can rebase your branch on the latest changes? Thanks!

**Comment by aslafy-z** [comment]
Will handle it since it's 2 years old now so we can merge it asap.

**Pull request closed without merging**